lib/cmd/clean.cc

```diff
@@ -0,0 +1,48 @@
+// std
+#include <iostream>
+#include <string>
+
+// external
+#include <spdlog/spdlog.h> // NOLINT(build/include_order)
+
+// internal
+#include "poac/cmd/clean.hpp"
+#include "poac/config.hpp"
+#include "poac/data/manifest.hpp"
+#include "poac/util/validator.hpp"
+
+namespace poac::cmd::clean {
+
+[[nodiscard]] Result<void>
+clean(const Options& opts) {
+  const auto profile =
+      Try(util::validator::valid_profile(opts.profile, opts.release)
+              .map_err(to_anyhow));
+  if (profile.has_value() && profile.value() != "debug"
+      && profile.value() != "release") {
+    return Err<build::UnsupportedProfile>(profile.value());
+  }
+
+  const auto path = profile.has_value()
```

Comment:
```suggestion
  const Path path = profile.has_value()
```

Let's use `Path` defined in the following:

https://github.com/poacpm/poac/blob/11d2a27c83aacde88d573e3307fd7cd2959c429a/include/poac/poac.hpp#L86

Reply:
✔️